### ISSUE TITLE
 Using graphql-playground module in create-react-app causes module not found errors?

### ISSUE THREAD

**Issue by StymiedSloth** [graphcool/graphql-playground#149]
I am trying to follow instructions to use the playground in a very simple project using create-react-app ... 

Commands I ran

```
create-react-app my-app
cd my-app
yarn add graphql-playground
yarn add isomorphic-fetch // just in case
yarn start
```

I import the module into my index file 
```import Playground from 'graphql-playground';```

I get the following Error
```
./src/index.js
Module not found: Can't resolve 'graphql-playground' in '~/Documents/dump/test/src'
```

I tried ```create-react-app my-app --scripts-version=react-scripts-ts``` too
and tried to add the module but still facing the same issue there.

My guess is there is some export missing in the code? I didn't spend too much time digging into this. Am I missing smthing? if anyone has faced the same issue, solutions/fixes are appreciated.

Thanks

**Comment by albinekb** [comment]
I'm getting the same errors when trying to import it in my next.js app 👍

**Comment by timsuchanek** [comment]
Thanks for reporting this! I just pushed a fixed to npm, version `1.0.4`. Feel free to reopen if this should still be an issue!

**Comment by StymiedSloth** [comment]
Thanks
